shell/store/prefs.js

```diff
@@ -523,28 +529,3 @@ export const actions = {
     }
   }
 };
-
-function getLoginRoute(route) {
```

Comment:
This was erroneously modifying the route name and omitting parameters. This meant only certain pages could be returned to but not all.